src/ThemeManager.cs

```diff
@@ -266,15 +275,21 @@ private static void DownloadMissingImages(List<ThemeConfig> missingThemes)
                 return;
             }
 
-            ProgressDialog downloadDialog = new ProgressDialog();

```

Comment:
Why was this line moved to the top of the class?

Reply:
Every time you click Retry, it would keep creating another instance of the progress dialog form.